.github/workflows/build-dockerfile.yml

```diff
@@ -0,0 +1,17 @@
+name: Build-Dockerfile
+
+on: pull_request
+
+jobs:
+  build-dockerfile:
+    runs-on: ubuntu-20.04
+
+    steps:
+      - name: Check out the repository
+        uses: actions/checkout@v2
+
+      - name: Build the Dockerfile
+        run: docker build -f examples/Dockerfile/Dockerfile -t tiledb:dev .
```

Comment:
The tag should be the the branch/tag name. This shouldn't be hard coded to `dev`. Sadly github doesn't offer the branch/tag name as an env so you'll need to get it from the ref: https://stackoverflow.com/questions/58033366/how-to-get-current-branch-within-github-actions .

Reply:
We should also align the dockerfile tag with the name to push to dockerhub.

Reply:
I agree -- I just tried to mimic the existing travis-ci that builds both a `dev` and `release`, although they are not published: https://github.com/TileDB-Inc/TileDB-Docker/blob/master/.travis.yml

Let's ticket this and return to it later when we approach automating the releases

Reply:
I've created clubhouse ticket 4767 to track this.